test/test_jit.py

```diff
@@ -4403,19 +4403,61 @@ def foo():
 
         self.checkScript(foo, ())
 
-    def test_string_sort(self):
-        @torch.jit.script
+    def test_string_sorted(self):
         def foo(strs: List[str]):
             return sorted(strs)
 
         FileCheck() \
             .check("graph") \
             .check_next("str[] = aten::sorted") \
             .check_next("return") \
-            .run(str(foo.graph))
+            .run(str(torch.jit.script(foo).graph))
+
+        inputs = ["str3", "str2", "str1"]
+        self.checkScript(foo, (inputs,))
+
+    def test_string_sort(self):
+        def foo(strs: List[str]):
+            strs.sort
```

Comment:
```suggestion
            strs.sort()
```

Reply:
More importantly, does it work? If so, why?

Reply:
Lol, it works because both scripted and unscripted version returns the unsorted string list. Fixed, also added a filecheck.